app/Http/Controllers/ParentController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\ParentModel;
+use App\Models\Category;
+use Illuminate\Http\Request;
+
+class ParentController extends Controller
+{
+    public function index(Request $request)
+    {
+        $search = $request->input('parentSearch');
```

Comment:
$request is een associative array dus het zou wss makkelijker zijn om te typen $request['parentSearch']